.github/workflows/delete_review_app_aks.yml

```diff
@@ -0,0 +1,43 @@
+name: Delete review app on AKS
+
+on:
+  pull_request:
+    branches:
+    - main
+    types:
+    - closed
+
+concurrency: deploy-${{ github.ref }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    environment: review
+
+    steps:
+    - name: Destroy review app
```

Comment:
name of this step should just be 'checkout' or similar

Reply:
Fixed this